### PR TITLE
Egao/my courses pinned tab

### PR DESCRIPTION
**Pull request by edwardg8**

### CODE REVIEW

File: src/d2l-my-courses-content/d2l-my-courses-behavior.html [hunk 1]
```diff
@@ -130,12 +135,23 @@
...
 					title: this.localize('allTab'),
 					selected: false,
 					enrollmentsSearchAction: this._enrollmentsSearchAction
-				}].concat(this._tabSearchActions);
+				}];
+				
+				if (this._pinnedTabAction){
+					actions = actions.concat({
+						name: this._pinnedTabAction.name,
+						title: this.localize('pinnedCourses'),
```
Comment (edwardg8) [inline src/d2l-my-courses-content/d2l-my-courses-behavior.html]: There is already "Pinned" langterm exist and not used anywhere, so I grabbed it here.

**Comment by njostonehouse** [comment]
> Tests passed locally

I think there's a linting error.

**Comment by edwardg8** [comment]
> > Tests passed locally
> 
> I think there's a linting error.

I saw that too, have we seen the linting error before? I kinda forgot what the tests failed on previous PRs

**Comment by edwardg8** [comment]
Sorry kinda ignored the errors from travis, yes there are space errors, will resolve those.

**Comment by edwardg8** [comment]
I did merge latest to this branch earlier, now have conflict with my local. Will close this one instead.

**Pull request closed without merging**